### PR TITLE
Fix odd back button behavior

### PR DESCRIPTION
**Pull request by MatthewSchultz**

### CODE REVIEW

File: app/forms/form_navigation.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ def initialize(controller)
   def next
     return unless index
 
-    controller_until_end = controllers[index + 1..-1]
+    controller_until_end = controllers[index + 1..]
```
Comment (luigi) [inline app/forms/form_navigation.rb:36]: What's the effect of making this range to be endless?

Comment (luigi) [reply app/forms/form_navigation.rb:36]: Answer: this is a (new?) style suggestion from Rubocop.

Comment (MatthewSchultz) [reply app/forms/form_navigation.rb:36]: IIRC, these two lines have the exact same effect....I don't think this changes anything programmatically...purely code style.

**Comment by MatthewSchultz** [comment]
@luigi - good catch, forgot my debug log.

**Pull request merged**